RevenueCatUI/Helpers/Package+VariableDataProvider.swift

```diff
@@ -41,7 +41,7 @@ extension Package: VariableDataProvider {
 
     func periodName(_ locale: Locale) -> String {
         return Localization.localized(packageType: self.packageType,
-                                      locale: locale)
+                                      locale: locale) ?? self.identifier
     }
```

Comment:
That would lead to duplicated code, but I agree it can be confusing. I'll add a commend.
I do think the implementation makes sense, this is used like: `package.periodName()`. So it's kind of an implementation detail that if the package type doesn't have a "period" we default to the package identifier.